the generator expressions get placed into the generated .pc file otherwise (non
x86/ppc)
diff --git a/cmake/Modules/CompilerConfig.cmake b/cmake/Modules/CompilerConfig.cmake
index c939146..89c022b 100644
--- a/cmake/Modules/CompilerConfig.cmake
+++ b/cmake/Modules/CompilerConfig.cmake
@@ -134,8 +134,7 @@ else()
     check_cxx_compiler_flag("-fopenmp-simd" CXX_COMPILER_SUPPORTS_OPENMP_SIMD)
     set(ARCH_SIMD_FLAGS
         -DSIMDE_ENABLE_OPENMP
-        "$<$<AND:$<COMPILE_LANGUAGE:C>,$<BOOL:C_COMPILER_SUPPORTS_OPENMP_SIMD>>:-fopenmp-simd>"
-        "$<$<AND:$<COMPILE_LANGUAGE:CXX>,$<BOOL:CXX_COMPILER_SUPPORTS_OPENMP_SIMD>>:-fopenmp-simd>"
+        -fopenmp-simd
     )
   endif()
 endif()
diff --git a/cmake/external/ObsPluginHelpers.cmake b/cmake/external/ObsPluginHelpers.cmake
index da2cdee..d547b01 100644
--- a/cmake/external/ObsPluginHelpers.cmake
+++ b/cmake/external/ObsPluginHelpers.cmake
@@ -272,8 +272,7 @@ elseif(_HOST_ARCH MATCHES "arm64|arm64e|aarch64")
       ${CMAKE_PROJECT_NAME}
       PRIVATE
         -DSIMDE_ENABLE_OPENMP
-        "$<$<AND:$<COMPILE_LANGUAGE:C>,$<BOOL:C_COMPILER_SUPPORTS_OPENMP_SIMD>>:-fopenmp-simd>"
-        "$<$<AND:$<COMPILE_LANGUAGE:CXX>,$<BOOL:CXX_COMPILER_SUPPORTS_OPENMP_SIMD>>:-fopenmp-simd>"
+        -fopenmp-simd
     )
   endif()
 endif()
